plugins/flashrom/fu-flashrom-lspcon-i2c-spi-device.c

```diff
@@ -64,9 +72,12 @@ fu_flashrom_lspcon_i2c_spi_device_probe (FuDevice *device, GError **error)
 	FuFlashromDevice *flashrom_device = FU_FLASHROM_DEVICE (device);
 	FuDeviceClass *klass =
 		FU_DEVICE_CLASS (fu_flashrom_lspcon_i2c_spi_device_parent_class);
-	g_autoptr(GRegex) regex = NULL;
-	g_autoptr(GMatchInfo) info = NULL;
-	const gchar *path = NULL;
+	g_autoptr(GUdevDevice) self_device = fu_udev_device_get_dev (FU_UDEV_DEVICE(device));
+	g_autoptr(GUdevDevice) self_parent_device = g_udev_device_get_parent (self_device);
+	const gchar *self_parent_path = g_udev_device_get_sysfs_path (self_parent_device);
+	g_autoptr(GUdevClient) udev_client = g_udev_client_new (NULL);
+	g_autoptr(GUdevEnumerator) udev_enumerator = g_udev_enumerator_new (udev_client);
```

Comment:
I think I would prefer to see direct usage of `GUdev` land in `fu-udev-device.c` as a helper function of some sort (maybe `fu_udev_get_sibling` with some arguments of what subsystem to find siblings of?)

Reply:
Yup, 100% agreed; all this needs to live in fu-udev-device.c

Reply:
Added fu_udev_device_get_siblings_with_subsystem.